src/mixins/filing-template-mixin.ts

```diff
@@ -85,6 +87,8 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
   @Action setProvisionsRemoved!: ActionBindingIF
   @Action setOriginalResolutionDates!: ActionBindingIF
   @Action setResolutionDates!: ActionBindingIF
+  @Action setCourtOrderNumber: ActionBindingIF
```

Comment:
Maybe the getter and setter names could be better synchronized, too.

Reply:
🤔 
I feel like Court order number is a consistent reference here within the UI. If we rename everything to be `fileNumber` it will be different from the UI components. 
one way or another there is a disconnect.

Reply:
OK, let's go with the lesser of 2 weevils 🐞 

(status quo)

Reply:
You've convinced me, will update all to fileNumber 😁 
In next commit!

Reply:
Updated 👍